src/shared_modules/rsync/src/rsyncImplementation.h

```diff
@@ -97,7 +98,8 @@ namespace RSync
 
             void releaseContext(const RSYNC_HANDLE handle);
 
-            RSYNC_HANDLE create();
+            RSYNC_HANDLE create(const unsigned int threadPoolSize = std::thread::hardware_concurrency(),
+                                const size_t maxQueueSize = 0);
```

Comment:
Hi @Dwordcito

Maybe the `UNLIMITED_QUEUE_SIZE` macro could be moved to a common file and used here as well.